t/030_ssl.t

```diff
@@ -8,15 +8,23 @@ use NAR::Helper;
 
 use Time::HiRes qw(gettimeofday tv_interval);
 
-my $helper = NAR::Helper->new(
-    ssl        => 1,
-    port       => 5673,
-    host       => 'rabbitmq.thisaintnews.com',
-    ssl_cacert => "$Bin/ssl/cacert.pem",
-    ssl_init   => 1,
-    username   => 'nartest',
-    password   => 'reallysecure',
-);
+if ($ENV{MQSKIPSSL}) {
+    plan skip_all => 'SSL tests disabled by user';
+} else {
+    plan tests => 10;
+}
+
+# MQSKIPSSL not set, set SSL-related options to default values unless
+# already set by the user.  Those ought to work with the default
+# server at rabbitmq.thisaintnews.com (which you get if you don't set
+# MQHOST).
+local $ENV{MQSSL} = 1;
```

Comment:
I kinda think that we should instantiate with env vars, not set env vars in out test.